### PR TITLE
Use Github Actions

### PR DESCRIPTION
**Pull request by jherdman**
This PR introduces GitHub Actions as a CI provider.

We will iterate on it, and refine it as we go!

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+          --health-interval 10s
+          --health-timeout 5s
+          --health-retries 5
+        ports:
+          - 5432:5432
+
+    strategy:
+      fail-fast: false
+      matrix:
+        ruby-version: [2.5.x, 2.6.x, 2.7.x]
```
Comment (jherdman) [inline .github/workflows/ci.yml:31]: NOTE: I've removed 2.4 as it's EOL https://www.ruby-lang.org/en/news/2020/04/05/support-of-ruby-2-4-has-ended/

**Comment by jherdman** [comment]
Unfortunately I can't iterate and test this until it's merged.

File: .github/workflows/ci.yml (hunk 1 above)
Comment (olleolleolle) [reply .github/workflows/ci.yml:31]: Thanks for the inspiration!

In order to make it formal, and _perhaps_ easier to loate, I'll make a small PR with a commit taking that version out, before we take this in.

(While doing that, I noticed ruby2.3 packages being used in Dockerfile. So, some spring cleaning was needed.

**Review by olleolleolle — APPROVED** [review]
LGTM!

Thank you so much for building this!

**Comment by olleolleolle** [comment]
@jesjos Do _you_ have to set some Setting, in order to make this run? And to retire Travis CI?

**Pull request merged**